inspirehep/modules/migrator/tasks.py

```diff
@@ -295,7 +295,7 @@ def migrate_chunk(chunk, broken_output=None, dry_run=False):
 
 
 @shared_task()
-def add_citation_counts(chunk_size=500, request_timeout=10):
+def add_citation_counts(chunk_size=500, request_timeout=40):
```

Comment:
Do you have a custom `Elasticsearch` config on your Honcho setup, or just running with defaults?